modules/simphera_base/keyvault.tf

```diff
@@ -26,7 +26,9 @@ resource "azurerm_key_vault" "simphera-key-vault" {
       "List",
       "Delete",
       "Update",
-      "Purge"
+      "Purge",
+      "Recover",
```

Comment:
Why are the `Recover` and `GetRotationPolicy` permissions now required?

Reply:
This is suggested in the upstream and on my test setup it produced errors (specifically because of lack of these permissions). So, I've consulted the official upstream docs (linked) and took default values.
After that it worked without error and no manual intervention is needed.

Reply:
And I've checked the same for v3.62.1 docs (old provider version), same suggestion in the upstream.